api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -247,7 +250,68 @@ void shouldUpdateClaimAndReturnClaimUpdatedForMatchingNinoWhenEligibleAndNoField
         verify(claimRepository).findById(existingClaimId);
         verify(claimRepository).save(result.getClaim());
         verify(eventAuditor).auditUpdatedClaim(result.getClaim(), emptyList());
-        verifyZeroInteractions(messageQueueDAO);
+        verifyZeroInteractions(messageQueueClient);
+    }
+
+    @Test
+    void shouldSendAdditionalPregnancyPaymentMessageWhenDueDateProvided() {
+        //given
+        Claimant existingClaimant = aClaimantWithExpectedDeliveryDate(null);
+        LocalDate expectedDeliveryDate = LocalDate.now().plusMonths(6);
+        Claimant newClaimant = aClaimantWithExpectedDeliveryDate(expectedDeliveryDate);
+        sendAdditionalPregnancyPaymentMessageTest(existingClaimant, newClaimant);
+    }
+
+    @Test
+    void shouldSendAdditionalPregnancyPaymentMessageWhenExistingDueDateUpdated() {
+        //given
+        Claimant existingClaimant = aClaimantWithExpectedDeliveryDate(LocalDate.now());
+        LocalDate expectedDeliveryDate = LocalDate.now().plusMonths(1);
+        Claimant newClaimant = aClaimantWithExpectedDeliveryDate(expectedDeliveryDate);
+        sendAdditionalPregnancyPaymentMessageTest(existingClaimant, newClaimant);
+    }
+
+    private void sendAdditionalPregnancyPaymentMessageTest(Claimant existingClaimant, Claimant newClaimant) {
```

Comment:
I think a better name for this would be `shouldSendAdditionalPregnancyPaymentMessage()`.

Reply:
Renamed.